packages/itwin/property-grid/package.json

```diff
@@ -29,28 +29,39 @@
     "copy:public": "cpx \"./public/**/*\" ./lib/public",
     "copy:cjs": "cpx \"./src/**/*.scss\" ./lib/cjs",
     "copy:esm": "cpx \"./src/**/*.scss\" ./lib/esm",
-    "cover": "",
+    "cover": "nyc npm run test",
```

Comment:
can you update the repository field, can look at how its done in the core repo now